lua/telescope/pickers.lua

```diff
@@ -493,6 +493,16 @@ function Picker:find()
           log.warn("Finder failed with msg: ", msg)
         end
 
+        -- add an nvim_win_set_cursor to force the results win be redrawn
+        vim.schedule(function()
+          if not vim.api.nvim_win_is_valid(self.results_win) then
+            return
+          end
+          vim.api.nvim_win_call(self.results_win, function()
+            vim.cmd "redraw!"
```

Comment:
Are you sure `!` is needed?

Otherwise this PR looks good.

Reply:
remove `!`:)